internal/context/context.go

```diff
@@ -35,6 +35,7 @@ var (
 	ctxRootModuleLoader  = &contextKey{"root module loader"}
 	ctxRootDir           = &contextKey{"root directory"}
 	ctxDiags             = &contextKey{"diagnostics"}
+	ctxServerID          = &contextKey{"server id"}
```

Comment:
Related to my comment below, given what we use this option for, would "command prefix" be a perhaps a better name for this?